ovs_performance.py

```diff
@@ -4141,6 +4161,10 @@ def main():
     if config.debug_dut_shell:
         dut_shell.logger.setLevel(logging.DEBUG)
 
+    # If new version dpdk-tool changes the command "testpmd" to "dpdk-testpmd"
+    # create soft link.
```

Comment:
Can you add the following line before the call?

    lprint("- Create testpmd link on VM for dpdk-testpmd, if needed...")

Reply:
Yes